### DISCUSSION TITLE
How to make it works with Ruby (Ruby on rails)

### DISCUSSION THREAD

**Discussion by SirMishaa** [spatie/ray#696] [Q&A]
Hello

I'd like to use ray on a rub on rails project, but it looks like that it doesn't work :(

It's forking for Laravel project and even nodejs project, but not in Ruby
I'm under WSL 2, Windows 11

I followed https://spatie.be/docs/ray/v1/installation-in-your-project/ruby

Should I make a configuration file? How with Ruby?

**Comment by freekmurze** [comment]
Unfortunately we don't have too much Ruby experience. 

I tried to make that gem when Ray was released, and it worked at that time.
Support for a config file was never added. I hope that at some point in the future someone from the Ruby / Rails community will step up and help us improve it.

**Comment by mewthu2** [comment]
hello dudes, follow up this steps :

https://gorails.com/setup/osx/11-big-sur (change os properties)

**Comment by diegoldev** [comment]
> Hello
> 
> I'd like to use ray on a rub on rails project, but it looks like that it doesn't work :(
> 
> It's forking for Laravel project and even nodejs project, but not in Ruby I'm under WSL 2, Windows 11
> 
> I followed https://spatie.be/docs/ray/v1/installation-in-your-project/ruby
> 
> Should I make a configuration file? How with Ruby?

Hi @SirMishaa !
Do you found any solve to this? I'm facing same issue.
The problem I find is that ray gem does not allow change configuration host as laravel package do. This is necessary in order to set the Windows Host IP as explain [here](https://myray.app/docs/environments/windows)

AFAIK the host is fixed on `ray.rb`  and can't be modified in any .env or config file.

![imagen](https://github.com/spatie/ray/assets/1553775/d17fb334-66b0-4eb6-9482-ee970bf39e98)

Is my assumpltion right @freekmurze ?

**Reply by SirMishaa** [reply]
@diegoldev 

Sorry, but I've completely stopped doing Ruby and Ruby on Rails, so I'm not able to answer.

**Reply by diegoldev** [reply]
No problem @SirMishaa !
I could find where was the issue.  It's necessary to be able to configure the ray host in gem but it has not a configuration file. I had to overwrite the source gem and it works
